lib/gocd/pipeline/pipeline_repository.rb

```diff
@@ -3,7 +3,7 @@
 module GOCD
   class PipelineRepository
     def self.pipelines
-      @pipelines = fetch_data
+      @pipelines ||= fetch_data
```

Comment:
Is this change intentional? Repo should cache the response and not make the call again.